pkg/tanka/workflow.go

```diff
@@ -185,5 +186,17 @@ func Show(baseDir string, opts Opts) (manifest.List, error) {
 // Eval returns the raw evaluated Jsonnet output (without any transformations)
 func Eval(dir string, opts Opts) (raw interface{}, err error) {
 	r, _, err := eval(dir, opts.JsonnetOpts)
-	return r, err
+	if err != nil {
+		switch err.(type) {
+		case ErrMultipleEnvs:
+			log.Println(err)
```

Comment:
I must admit I can't quite follow this code. To me this seems as encountering multiple environments is no longer a fatal thing, but only a warning, given no error is returned.

I'd expect `Eval` to behave like so:

- return an `error` everytime an error occured (e.g. invalid Jsonnet, no Environment, multiple Environments, etc)
- as you need to handle the "no inline but maybe spec.json" case further down the road, this specific case should also return data

This allows a naive callee to automatically abort when any error occurs. Callees aware of inline vs spec.json can react to receiving `ErrNoEnv`

Reply:
That is what I did before, simply return both data and err from upstream to downstream, no handling here.

Reply:
Sure, but this is different from what I just described.

Only the specific case of "no inline environment" has a reason to return data alongside an error. Afaict, all other errors that occur here are fatal and thus return `nil, err`

Reply:
Okay, so here you say return data+err? Gotcha.

Reply:
Yes, which you could simplify along those lines:

```go
// Eval returns the raw evaluated Jsonnet output (without any transformations)
func Eval(dir string, opts Opts) (raw interface{}, err error) {
	r, _, err := eval(dir, opts.JsonnetOpts)
        if errors.Is(err, ErrNoEnv) {
                return r, err
        } else if err != nil {
                return nil, err
        }

	return r, nil
}
```